src/cmd/index.js

```diff
@@ -3,5 +3,6 @@ import build from './build';
 import lint from './lint';
 import run from './run';
 import sign from './sign';
+import newCommand from './new';
 
-export default {build, lint, run, sign};
+export default {build, lint, run, sign, newCommand};
```

Comment:
I think renaming the command is the best solution since it will allow us to match the name of the function to the command exactly.

Here are some ideas along with how they might be mnemonically associated in the developer's mind:

* `web-ext create`
  * "Create a new web extension"
* `web-ext start`
  * "Start developing a new web extension"
  * One downside is that `npm start` is a convention which usually applies more to starting a server or a test suite, depending on the project.
* `web-ext init`
  * "Initialize a new web extension"
  * I'm not totally opposed to this, it just seems like an unnecessary abbreviation to me when we have options for shorter words.

Any of these are fine by me so pick your favorite :) I would probably choose `create` myself.